readme.md

```diff
@@ -152,38 +152,50 @@ If element names start with an uppercase letter, `dom-chef` will consider them a
 ```jsx
 function Title() {
 	const title = document.createElement('h1');
-	title.classList.add('Heading');
+	title.classList.add('Heading', 'red');
+	title.append('La Divina Commedia');
 	return title;
 }
 
-const el = <Title className="red">La Divina Commedia</Title>;
+const el = <Title/>;
 // <h1 class="Heading red">La Divina Commedia</h1>
 ```
 
-This makes JSX also a great way to apply multiple attributes and content at once:
-
 ```jsx
-const BaseIcon = () => document.querySelector('svg.icon').cloneNode(true);
+function Title(props) {
+	const title = <h1 {...props}></h1>;
+	title.classList.add('red');
+	return title;
+}
 
-document.body.append(
-	<BaseIcon width="16" title="Starry Day" className="margin-0" />
-);
+const el = <Title className="Heading">La Divina Commedia</Title>;
+// <h1 class="Heading red">La Divina Commedia</h1>
 ```
 
-To improve compatibility with React components, `dom-chef` will pass the function's `defaultProps` property to itself (if present). Note that specifying attributes won't override those defaults, but instead set them on the resulting element:
+To improve compatibility with React components, `dom-chef` will pass the function's `defaultProps` property to itself (if present). Note that specifying attributes will override those defaults:
```

Comment:
Given that this now follows React’s behavior it doesn't need to be explained:

```suggestion
[`defaultProps`](https://medium.com/@pojotorshemi/react-basis-default-props-and-typechecking-with-proptypes-4ddf02d15992) is also supported and will be included as props to the function as you'd expect:
```

However I can't find an official documentation for it. Do you know a better resource?